nxtool/log_providers/elastic.py

```diff
@@ -18,28 +18,38 @@
 
 from elasticsearch import TransportError
 from elasticsearch_dsl import Search, Q
-from elasticsearch_dsl import DocType, Date, Boolean, String, Integer, Ip, GeoPoint
+from elasticsearch_dsl import DocType, Date, Boolean, Integer, Ip, GeoPoint
+from elasticsearch_dsl import Index, VERSION
 from elasticsearch_dsl.connections import connections
 
+try:
+    from elasticsearch_dsl import Text, Keyword
+except ImportError:  # oldversion of Elasticsearch
+    from elasticsearch_dsl import String as Text
+    from elasticsearch_dsl import String as Keyword
```

Comment:
I guess a more readable way to write it would be:

```python
from elasticsearch_dsl import String as Text
String = Text
```

Since currently, one might think that it's a typo.